src/test/java/io/cdap/plugin/successfactors/connector/SuccessFactorsConnectorTest.java

```diff
@@ -85,8 +85,7 @@ public void testConfiguration() throws TransportException, SuccessFactorsService
 
   @Test
   public void testValidateSuccessfulConnection() throws TransportException, SuccessFactorsServiceException {
-    successFactorsTransporter = new SuccessFactorsTransporter(pluginConfig.getConnection().getUsername(),
-                                                              pluginConfig.getConnection().getPassword());
+    successFactorsTransporter = new SuccessFactorsTransporter(pluginConfig.getConnection());
```

Comment:
these initializations can be done once in the `@Before` method as this instance is same throughout all the tests for cleaner code.

Reply:
Done